test/shopify-cli/changelog_test.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+require "test_helper"
+require "shopify_cli/changelog"
+
+module ShopifyCLI
+  class ChangelogTest < MiniTest::Test
+    def test_changelog_outputs_equal_inputs
+      assert_equal(Changelog.new.full_contents, File.read(Changelog::CHANGELOG_FILE))
```

Comment:
This test is really important! It asserts that, without any changes, the parser will output the exact file char-for-char that it took as input. So we can be sure that it's respecting whatever we've done in the past, even when a change is recorded.